picard/ui/metadatabox.py

```diff
@@ -317,13 +329,25 @@ def contextMenuEvent(self, event):
         menu.exec_(event.globalPos())
         event.accept()
 
+    @staticmethod
+    def _save_preserved_tags(preserved_tags):
+        config.setting['preserved_tags'] = ", ".join(uniqify(preserved_tags))
+
+    def add_to_preserved_tags(self, name, preserved_tags):
+        preserved_tags.append(name)
+        self._save_preserved_tags(preserved_tags)
+
+    def remove_from_preserved_tags(self, name, preserved_tags):
+        preserved_tags = filter(lambda x: x != name, preserved_tags)
```

Comment:
Did not do that in case there are multiple instances of the name ( maybe entered by the user)